src/Adapter/Product/AdminProductDataProvider.php

```diff
@@ -379,4 +379,12 @@ public function getPaginationLimitChoices()
 
         return $paginationLimitChoices;
     }
+
+    /* (non-PHPdoc)
```

Comment:
To ensure PHPdocumentor make no error on these, I just replaced these 'non php-doc' comments by {@inheritdoc}, on another PR.